repo/tuf_on_ci/_repository.py

```diff
@@ -110,6 +113,10 @@ def __init__(self, dir: str, prev_dir: str | None = None):
 
         # read signing event state file
         self.state = SigningEventState(os.path.join(self._dir, ".signing-event-state"))
+    """Maximum depth within role directory. This
+    should be the same as SignerRepository.MaxDepth
+    in the signer project"""
+    MaxDepth: int = 4
```

Comment:
similar to other, constants are CAPS. Unless you want this configurable then it should be a member variable in the constructor with a default. 
```suggestion
    # Maximum depth within role directory
    MAX_DEPTH = 4 # should match signer.SignerRepository.MAX_DEPTH
```

Reply:
it would be better to have this be in a configuration module that both modules import
like
```python
from tuf_on_ci import config
config.MAX_TARGETS_DEPTH
```